.github/workflows/prune.yaml

```diff
@@ -0,0 +1,36 @@
+# https://docs.github.com/en/actions
+
+name: "Prune"
+
+on:
+    schedule:
+        - cron: "0 12 * * *"
+
+env:
+    DAYS_BEFORE_CLOSE: 30
+    DAYS_BEFORE_STALE: 365
+
+jobs:
+    prune:
+        name: "Issues"
+
+        runs-on: "ubuntu-latest"
+
+        steps:
+            -
+                name: "Prune stale issues and pull requests"
+                uses: "actions/stale@v3"
+                with:
+                    days-before-close: ${{ env.DAYS_BEFORE_CLOSE }}
+                    days-before-stale: ${{ env.DAYS_BEFORE_STALE }}
+                    repo-token: "${{ secrets.GITHUB_TOKEN }}"
+                    stale-issue-label: "stale"
```

Comment:
Do we have a `stale` label, or something similar?

If so, we need to reference the label name here.

Reply:
what do you mean by "reference"? like example you did, or some github-label-ID ?

Reply:
@keradus 

We need to reference the name of the label here.

Reply:
@keradus 

Can you create a label, perhaps `stale` or `status/stale`? I do not have privileges to create a label.